bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -144,7 +144,7 @@ invalid.retry.count = invalid retry count
 arguments.count.mismatch = arguments count mismatch: expected {0}, found {1}
 not.enough.format.arguments = not enough format arguments
 invalid.format.specifier = unknown format conversion ''{0}''
-invalid.map.insertion = invalid map insertion: expected value of type ''{0}'', found ''{1}''
+invalid.map.insertion = invalid map insertion, expected value of type ''{0}'', found ''{1}''
```

Comment:
The previous format is what we've used normally. Any particular reason to change it here?

Reply:
With this change when the Ballerina program panics, it will print something like `error: Inherent type violation: invalid map insertion, expected value of type 'byte', found 'int'`

Without this change, it will print `error: Inherent type violation: invalid map insertion: the expected value of type 'byte', found 'int'` 
I think having two `:` in the error message looks weird here.

Actually I have a few issues I'd like to discuss
1. InherentTypeViolation was the error reason, since we don't have error reasons now, do we need to somehow embed error reason in the error message.
2. Should the error message be capitalized (First letter)

Reply:
Ah I see. Yeah, multiple `:`s do look weird. May have to rephrase these.

Reply:
@MaryamZi

Reply:
How about something like

`error: inherent type violating map insertion: expected value of type 'byte', found 'int'`

And I believe we decided to start with a lowercase letter, that's the behaviour before the error change right?

Reply:
The message part of the error detail used to start with lower case letters, but If we look at the full error message (after .toString()) It usually starts with the Upper case error reason tag ({module/name}TheErrorReason ...)
Like this: https://github.com/ballerina-platform/ballerina-lang/pull/24683/files#diff-76b0195bf9aa795ecdba0679ce1aa184L175
Will schedule a meeting to discuss in detail about appropriate error messages